www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/Api/DiagnosticsController.php

```diff
@@ -76,14 +76,9 @@ public function caddyfileAction()
         // Decode JSON to PHP array
         $responseArray = json_decode($response, true);
 
+        // Since errors are handled by the caddy_diagnostics script and returned as json, check for an error key in the response
         if (isset($responseArray['error'])) {
-            // Handle the error
-            return ["status" => "failed", "message" => $responseArray['message']];
-        }
-
-        // Assuming the response structure is like { "content": "actual Caddyfile content here" }
-        if (!isset($responseArray['content'])) {
-            return ["status" => "failed", "message" => "Caddyfile content not found"];
+            return ["status" => "failed", "message" => ($responseArray['message']];
```

Comment:
There's a stray "(" there that would likely cause a syntax error?

Reply:
I made a mistake while editing the gettext function out. Thanks for catching that.